app/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
         tools:ignore="QueryAllPackagesPermission" />
     <uses-permission android:name="com.android.alarm.permission.SET_ALARM" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="android.permission.REQUEST_COMPANION_START_FOREGROUND_SERVICES_FROM_BACKGROUND"/>
```

Comment:
I believe this is only for using the [Companion Device Manager](https://developer.android.com/develop/background-work/services/foreground-services#background-start-restriction-exemptions) (for example smartwatch apps on the phone), which HA does not use or qualify for

Reply:
Yeah I think it was added when I was trying to get the service to work.